src/shapes/IText/ITextBehavior.ts

```diff
@@ -917,6 +920,11 @@ export abstract class ITextBehavior<
     start: number,
     copiedStyle?: TextStyleDeclaration[]
   ) {
+    if (copiedStyle && insertedText.length !== copiedStyle.length) {
+      throw new Error(
+        `fabric.js: expected ${insertedText.length} style declarations, found ${copiedStyle.length}`
+      );
+    }
```

Comment:
This seems something to cover for developers that decide to mess with te copy/paste event and i don't want to do it.
is there a legit reason for this to happen?

Reply:
A left over. Can be removed.
Was heard to understand what was going on without it.
But still that is what the method expects and if it receives something else it doesn't throw but the text object will be a mess